docs/developers/relayed-transactions.md

```diff
@@ -168,7 +168,7 @@ the matching gas limit values between the relayed and inner transactions.
 
 It would look like:
 
-```rust
+```
```

Comment:
the `rust` was needed for a better display of the code block. does it look ok now?

Reply:
Un-did, thanks!